### PR TITLE
net-config

### PR DESCRIPTION
**Pull request by henderiw**
net-config takes the ip from the network container and updates the template so the ip address is known to connect and apply the config to the device

### CODE REVIEW

File: e2e/tests/003-network-topo.tmpl [hunk 1]
```diff
@@ -5,6 +5,7 @@ metadata:
 spec:
   nodes:
     srl: 
+      address: $LEAF_IP:57400
       provider: srl.nokia.com
```
Comment (vjayaramrh) [inline e2e/tests/003-network-topo.tmpl:9]: curious if this is something nokia specific?

Comment (henderiw) [reply e2e/tests/003-network-topo.tmpl:9]: no this is relevant for any network device. but the port might be different

Comment (vjayaramrh) [reply e2e/tests/003-network-topo.tmpl:9]: > no this is relevant for any network device. but the port might be different

Apologies, I mean the `provider: srl.nokia.com`

**Review by vjayaramrh — APPROVED** [review]
reviewed, but not tested.
/lgtm

**Comment by henderiw** [comment]
no the idea is to be able to support various devices and the provider allows to select the one we connect to

File: e2e/tests/003-network-topo.tmpl (hunk 1 above)
Comment (radoslawc) [reply e2e/tests/003-network-topo.tmpl:9]: @henderiw 
```
provider: srl.nokia.com
```
That I guess should be changed to something more generic if that needs to be there as an configuration example as @vjayaramrh pointed out

**Pull request merged**